src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -304,11 +304,26 @@ Below is a list of available sub-commands:
 <!-- markdownlint-disable line-length -->
 | Flag                             | Description
 |:---------------------------------|:----------------------------------------------------------------------------|
-| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `default` namespace is used
+| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `submariner-operator` namespace is used
 | `--validation-timeout` `<value>` | Timeout in seconds while validating the connection attempt
 | `--verbose`                      | Produce verbose logs during validation
 <!-- markdownlint-enable line-length -->
 
+{{% notice warning %}}
+In Kubernetes 1.23 and above, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
```

Comment:
```suggestion
Starting with Kubernetes 1.23, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
```

Reply:
@dfarrell07 probably can make a more grammatically correct suggestion :)

Reply:
This looks good to me :+1: